docs/recipes/integration-with-sentry.md

```diff
@@ -0,0 +1,119 @@
+# Integration with Sentry
+
+If you are using Sentry as your Error Tracker, you can use this recipe to trace and log actions as *breadcrumbs*.
+
+First you'll need to initialize Sentry in your app using `Sentry.init(...)` in the `main.ts` file.
+
+```ts
+Sentry.init({
+  dsn: ...
+  // ...
+});
+``` 
+
+Next, you need to provide `Sentry`'s `ErrorHandler` and `TraceService`.
+
+```ts
+// app.module.ts
+@NgModule({
+  providers: [
+    // ...
+    {
+      provide: ErrorHandler,
+      useValue: Sentry.createErrorHandler(),
+    },
+    {
+      provide: Sentry.TraceService,
+      deps: [Router],
+    },
+    {
+      provide: APP_INITIALIZER,
+      useFactory: () => () => {},
+      deps: [Sentry.TraceService],
+      multi: true,
+    },
+  ],  
+})
+export class AppModule {}
+
+//app.config.ts
+export const appConfig: ApplicationConfig = {
+  providers: [
+    // ...
+    {
+      provide: ErrorHandler,
+      useValue: Sentry.createErrorHandler(),
+    },
+    {
+      provide: Sentry.TraceService,
+      deps: [Router],
+    },
+    {
+      provide: APP_INITIALIZER,
+      useFactory: () => () => {},
+      deps: [Sentry.TraceService],
+      multi: true,
+    },
+  ],
+};
+```
+
+Now, let's go over how to log NGXS actions as breadcrumbs.
+Create a service `NgxsSentryBreadcrumbsService`. 
+This service uses NGXS `Actions` dependendecy to listen to all events and send them to Sentry using `Sentry.addBreadcrumb`.
+
+```ts
+@Injectable()
+export class NgxsSentryBreadcrumbsService implements OnDestroy {
+  #actions = inject(Actions);
+  #destroyed$ = new Subject<void>();
+
+  constructor() {
+    this.#subscribeToActions();
+  }
+
+  #subscribeToActions() {
+    const excludeAction = [] //You can add the actions you want to exclude from your breadcrumbs here;
+    this.#actions
+      .pipe(
+        tap((ctx) => {
```

Comment:
The `tap` body can be moved to subscribe block as thus the operator would be removed.